napari/_qt/qt_main_window.py

```diff
@@ -627,7 +638,9 @@ def _add_menus(self):
 
         self.file_menu = menus.FileMenu(self)
         self.main_menu.addMenu(self.file_menu)
-        self.view_menu = menus.ViewMenu(self)
+        self.view_menu = build_qmodel_menu(
+            MenuId.MENUBAR_VIEW, title=trans._('View'), parent=self._qt_window
```

Comment:
@lucyleeow Can you fix? 

Good catch observing the View menu is lacking the accelerator shortcut. This will need to get fixed imo. I think the suggestion below might fix it, but I don't have time to test.

```suggestion
            MenuId.MENUBAR_VIEW, title=trans._('&View'), parent=self._qt_window
```

Reply:
Done and checked that it works